lib/fluent/plugin/output.rb

```diff
@@ -378,6 +377,7 @@ def configure(conf)
           buffer_conf = conf.elements(name: 'buffer').first || Fluent::Config::Element.new('buffer', '', {}, [])
           @buffer = Plugin.new_buffer(buffer_type, parent: self)
           @buffer.configure(buffer_conf)
+          keep_buffer_config_compat
```

Comment:
I'm considering how to do it more gracefully, but I don't yet get good idea.
I'll merge this as is, after considering a few days...

Reply:
I see...

 Is there a possibility that it would be better not to transfer the option to `Buffer`?